### PR TITLE
Create next steps page

### LINKED ISSUE #91413: Next Steps page
## Issue Description 

The Next Steps page instructs users to download, print and sign their form.  The Next Steps page will be available to **all users**.

**Dynamic content**: If the user selects a **VSO** for representation, they will see "VA Form 21-22" at the top of the page. If the user selects an **attorney/claims agent** for representation, they will see "VA Form 21-22a" at the top of the page.

**URL**: `/next-steps`

**Previous path**: 
1. ~`/download`~ `/confirmation`

**Next paths**: None!  This is the end!

[MVP Planning Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1714785369605/65b1ce20d08478615fe2ef8c347cf19305cbffab?wid=4-1714808680763)

## Designs
1. [Mobile Prototype](https://www.figma.com/proto/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=4311-279442&t=7ShW8oqmmrjY8jRD-0&scaling=min-zoom&content-scaling=fixed&page-id=455%3A76935&starting-point-node-id=456%3A171489&show-proto-sidebar=1)
2. [Canvas](https://www.figma.com/design/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=4311-279442&t=7ShW8oqmmrjY8jRD-4)
3. [Desktop Annotations](https://www.figma.com/design/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=5373-210712&t=7ShW8oqmmrjY8jRD-4) 

---
## Tasks
- [x] Create the Next Steps page based on the designs above
- [x] Set up the proper logic for displaying the page, as well as the content within
- [x] Set up routing to the proper form fields

## Acceptance Criteria
- [ ] A Next Steps page has been implemented, with the specs outlined above.

### PR DESCRIPTION
**Pull request by holdenhinkle**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/91413
Resolves comments in https://github.com/department-of-veterans-affairs/va.gov-team/issues/91412

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
- Created route to /next-steps
- Created /next-steps page (`NextStepsPage`)
- Fixed minor formation issue on /confirmation page
- Wrote unit tests for the following pages/components
  - `ConfirmationPage`
  - `AddressBlock`
  - `ContactCard`
  - `Email`
  - `GoogleMapLink`
  - `ParsePhoneNumber`
  - `Phone`

## Related issue(s)
- Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/91413
- Resolves comments in https://github.com/department-of-veterans-affairs/va.gov-team/issues/91412

## Testing done
- New unit tests added

### Changed spacing and added the missing Need Help? section:
image

### 21-22 With VSO Representative
image

### 21-22 Without VSO Representative
image

### 21-22a
image

## What areas of the site does it impact?
- Appoint a Rep tool which has not launched yet (it's behind a feature flag)

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication
NA
- [ ] ~Did you login to a local build and verify all authenticated routes work as expected with a test user~

## Requested Feedback
NA

### CODE REVIEW

File: src/applications/representative-appoint/containers/NextStepsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      return null;
+    }
+
+    const id = formData?.selectedAccreditedOrganizationId;
+    const orgs =
+      formData['view:selectedRepresentative']?.attributes
+        .accreditedOrganizations.data;
+    let orgName = null;
+
+    if (id && orgs) {
```
Comment (jvcAdHoc) [inline src/applications/representative-appoint/containers/NextStepsPage.jsx:55]: You should be able to use the `.find` array method

**Review by powellkerry — DISMISSED** [review]

File: src/applications/representative-appoint/containers/NextStepsPage.jsx (hunk 1 above)
Comment (holdenhinkle) [reply src/applications/representative-appoint/containers/NextStepsPage.jsx:60]: Nice. Updated.

**Review by cosu419 — DISMISSED** [review]
Approved

**Review by cosu419 — DISMISSED** [review]

**Review by powellkerry — DISMISSED** [review]

**Review by powellkerry — APPROVED** [review]

File: src/applications/representative-appoint/containers/NextStepsPage.jsx (hunk 1 above)
Comment (holdenhinkle) [reply src/applications/representative-appoint/containers/NextStepsPage.jsx:60]: Thanks, I think I prefer doing it this way.

**Pull request merged**